.github/workflows/backport-assistant.yml

```diff
@@ -0,0 +1,28 @@
+# This action creates downstream PRs for PRs with backport labels defined.
+# See docs here: https://github.com/hashicorp/backport-assistant
+
+name: Backport Assistant Runner
+
+on:
+  # TODO: Figure out if this should be `pull_request_target` if we have forks.
+  pull_request:
```

Comment:
I think we do want `pull_request_target` here. From the docs for `pull_request_target`:
> This event runs in the context of the base of the pull request, rather than in the context of the merge commit. This prevents execution of unsafe code from the head of the pull request that could alter your repository or steal any secrets you use in your workflow.

Also, looking at the other users of `backport-assistant` ([query](https://cs.github.com/?scopeName=All+repos&scope=&q=+hashicorpdev%2Fbackport-assistant+path%3A.github)), the majority use `pull_request_target`.

Reply:
Oh, that's a much clearer description that the backport-assistant docs. I don't think it's too much of an issue since this only runs on merge but `pull_request_target` is the better choice.